### PR TITLE
Fish 6301 "Enable Asadmin Recorder" button is not visible

### PR DESCRIPTION
**Pull request by luiseufrasio**
## Description
The "Enable Asadmin Recorder" button is not visible in the header of the Admin Console

### Blockers
None

### New tests
None

### Testing Performed

1. mvn clean install -DskipTests
2. ./appserver/distributions/payara/target/stage/payara6/bin/asadmin start-domain
3. Open localhost:4848 in a web browser
4. Check if the button appears

### Testing Environment
Zulu JDK 11 on Ubuntu 18.04 with Maven 3.8.4

### CODE REVIEW

**Comment by luiseufrasio** [comment]
jenkins test please

File: appserver/admingui/payara-theme/src/main/resources/branding/masthead.inc [hunk 1]
```diff
@@ -94,26 +112,4 @@
             />
         </sun:panelGroup>
     </facet>
-    <!facet userInfoExtras>
```
Comment (MarkWareham) [inline appserver/admingui/payara-theme/src/main/resources/branding/masthead.inc:97]: Looks like you removed line 97, which is perhaps an attempt at an opening tag? (I'm not familiar with woodstock) but not removed the corresponding close tag

Comment (MarkWareham) [reply appserver/admingui/payara-theme/src/main/resources/branding/masthead.inc:97]: nevermind it was the split view in github confusing matters

**Review by arieki — APPROVED** [review]
LGTM

**Pull request merged**